### DISCUSSION TITLE
Regarding Vendor Defined Message

### DISCUSSION THREAD

**Discussion by PoojaGaur1708** [DMTF/libspdm#847] [Q&A] [answered]
Hello , i want to send vendor defined request. 
Can you pls tell me if this message is supported in current libspdm package.
If yes , pls share any reference code that i can follow.

**Comment by jyao1** [comment, marked as answer]
Please refer to https://github.com/DMTF/spdm-emu/blob/main/library/pci_doe_requester_lib/pci_doe_spdm_vendor_send_receive.c#L28, 'pci_doe_spdm_vendor_send_receive_data() '
